addons/survey/__manifest__.py

```diff
@@ -22,6 +22,8 @@
         'mail',
         'web_tour'],
     'data': [
+        'views/report_certification.xml',
```

Comment:
`survey_report_templates.xml` and `survey_reports.xml` to fit guidelines. No need to specify 'certification' in filename as we don't have many reports for now ;)